modules/demo/src/main/scala/lucuma/ui/demo/Demo.scala

```diff
@@ -17,23 +18,57 @@ import japgolly.scalajs.react.vdom.html_<^._
 import lucuma.ui.forms._
 import monocle.macros.Lenses
 import org.scalajs.dom
+import _root_.react.semanticui.collections.form.Form
```

Comment:
Why `_root_`?

Reply:
`crystal` has a `react` package, so since there's an `import crystal._`, VSCode/Metals added the suggested import for `Form` like this. I am moving the `Form` import before `import crystal._` so we can remove `_root_`.